src/foundry/common/abstract/document.d.mts

```diff
@@ -891,7 +891,7 @@ declare abstract class Document<
 
   /**
    * @deprecated since v12, will be removed in v14
-   * @remarks `"The Document._onDeleteDocuments static method is deprecated in favor of Document._onDeleteOperation"`
+   * @remarks "The `Document._onDeleteDocuments` static method is deprecated in favor of {@link Document._onDeleteOperation | `Document._onDeleteOperation`}"
```

Comment:
Nit; you can use `@linkcode`. I suppose you may be avoiding that due to not having the TSDoc change in this PR?